### PR TITLE
Added Order page

### PR DESCRIPTION
**Pull request by SnehaGhadge1**
@snehaghadge70 

please review below code change.

1. added order page class
2. added order Test class

### CODE REVIEW

File: src/Pages/OrderPage.java [hunk 1]
```diff
@@ -0,0 +1,8 @@
+package Pages;
+
+public class OrderPage {
+
+	public void order() {
```
Comment (snehaghadge70) [inline src/Pages/OrderPage.java:5]: where is logic. please add logic.

File: src/Tests/OrderTest.java [hunk 2]
```diff
@@ -0,0 +1,8 @@
+package Tests;
+
+public class OrderTest {
+
+	int price = 888;
+	String prodName = "TV";
+	String customer = "naveen";
```
Comment (snehaghadge70) [inline src/Tests/OrderTest.java:7]: Add model number of the product.

**Review by snehaghadge70 — CHANGES_REQUESTED** [review]
Hi, 

please make the changes as per review commnets

File: src/Pages/OrderPage.java (hunk 1 above)
Comment (SnehaGhadge1) [reply src/Pages/OrderPage.java:5]: done. logic added.

File: src/Tests/OrderTest.java (hunk 2 above)
Comment (SnehaGhadge1) [reply src/Tests/OrderTest.java:7]: done. Added model number of the product.

**Review by snehaghadge70 — APPROVED** [review]
Review Approved

**Comment by SnehaGhadge1** [comment]
Adding order page class  and test class in Master

**Pull request merged**